model/label.rb

```diff
@@ -10,6 +11,6 @@ def initialize(title, color, id = Random.rand(1..999_999))
 
   def add_item(item)
     @items << item
-    item.label = self
+    # item.label = self
```

Comment:
It's better to remove your comments and it makes your code cleaner. If you needed the previous version of your code,  you could use git commit history instead of inline comments.